src/components/Forms/Textarea.jsx

```diff
@@ -37,6 +37,7 @@ const Textarea = ({
         name={name}
         placeholder={placeholder}
         value={value}
+        style={{ resize: "none", height: "50px" }}
```

Comment:
tailwind only

Reply:
is this from the previous PR? make sure to separate it

Reply:
yeah sorry it was from the other PR but it ended up here loll but i deleted it and im gonna make sure the resize changes are in the other branch